_test/test_sqw_file/test_head.m

```diff
@@ -29,23 +29,51 @@ function setUp(~)
         function tearDown(~)
             warning('on','SQW_FILE_IO:legacy_data');
         end
-
+        function test_display_sample(obj)
+            % redirect output to text string
+            text = evalc('head(obj.files{1})');
+            strings = splitlines(text);
+            ref_text = {' ';'';...
+                '  4-dimensional object:';...
+                ' -------------------------';...
+                ' Original datafile:  path to: test_sqw_file_read_write_v3_1.sqw';...
+                '             Title: <none>';...
+                '';' Lattice parameters (Angstroms and degrees):';...
+                '     a=4              b=5               c=6          ';...
+                ' alpha=91          beta=92          gamma=93         ';...
+                '';' Extent of data:';...
+                ' Number of spe files: 1';...
+                '    Number of pixels: 7680';...
+                '';' Size of 4-dimensional dataset: [5x5x5x5]';...
+                '     Plot axes:';...
+                '          Q_\zeta = -0.81977:0.40721:1.2163 in [Q_\zeta, 0, 0]';...
+                '          Q_\xi = -0.32672:0.32652:1.3059 in [0, Q_\xi, 0]';...
+                '          Q_\eta = -2.5103:0.36311:-0.69474 in [0, 0, Q_\eta]';...
+                '          E = -9.75:7.9:29.75';...
+                ' Object is stored in Horace-3.1 format file';...
+                '';''};
```

Comment:
The way this is formatted is very unclear to me. Matlab reads newlines in array constructors as `;`, so this could be:
```suggestion
            ref_text = {' '
                        ''
                        '  4-dimensional object:'
                        ' -------------------------'
                        ' Original datafile:  path to: test_sqw_file_read_write_v3_1.sqw'
                        '             Title: <none>'
                        ''
                        ' Lattice parameters (Angstroms and degrees):'
                        '     a=4              b=5               c=6          '
                        ' alpha=91          beta=92          gamma=93         '
                        ''
                        ' Extent of data:'
                        ' Number of spe files: 1'
                        '    Number of pixels: 7680'
                        ''
                        ' Size of 4-dimensional dataset: [5x5x5x5]'
                        '     Plot axes:'
                        '          Q_\zeta = -0.81977:0.40721:1.2163 in [Q_\zeta, 0, 0]'
                        '          Q_\xi = -0.32672:0.32652:1.3059 in [0, Q_\xi, 0]'
                        '          Q_\eta = -2.5103:0.36311:-0.69474 in [0, 0, Q_\eta]'
                        '          E = -9.75:7.9:29.75'
                        ' Object is stored in Horace-3.1 format file'
                        ''
                        ''}
```
Which actually looks like the text.